.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```
@@ -0,0 +1,108 @@
+
+
+          If the issue is render performance-related, try to reproduce it with `--no-config
+          --profile=fast`, which is designed to work on lower-end mobile devices.
+
+          For decoding performance-related issues, additionally try to enable hardware
+          decoding with `--hwdec=yes` or the `Ctrl+h` shortcut.
+
+
+          Describe the reproduction steps as precise as possible. It's very likely that
+          the bug you experience wasn't reproduced by the developer because the workflow
+          differs from your own.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Expected Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Actual Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Log File"
+      description: >
+          Make a log file with `--log-file=output.txt`, which is our preferred way, and attach it to
+          the issue.
+
+
+          If you have trouble producing a log file, you can alternatively use `-v -v` and save the
+          terminal output to a file.
+
+
+          In the case of a crash, please provide a backtrace.
+
+
+          Without the log file, this issue will be closed for ignoring the issue template.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Sample Files"
+      description: >
+          Please provide sample files for a minimal reproduction case, if the issue can't be
+          reproduced without it or it being helpful to demonstrate the issue.
+
+
+          Files can be attached directly within the text field below. If github's upload restriction
+          (file type, size, etc) don't allow the direct upload, use a solutions that is easy to
+          access with possible long-term retention. Possible service can be but are not limited to:
+
+          - [0x0.st](https://0x0.st/) (no long-term retention)
+           - [Dropbox](https://www.dropbox.com)
+           - [Google Drive](https://www.google.com/drive/)
+           - [iCLoud Drive](https://www.icloud.com/iclouddrive/)
+           - [OneDrive](https://onedrive.live.com/)
+- type: checkboxes
+  attributes:
+      label: "I carefully read all instruction and confirm that i did the following"
+      options:
+          - label: >-
+              I produced the log file with the exact same set of files, parameters, and conditions
+              used in "Reproduction Steps", with the addition of `--log-file=output.txt`.
+            required: true
+          - label: >-
+              I produced the log file while the behaviors described in "Actual Behavior" were
+              actively observed.
+            required: true
+          - label: "I attached the full, untruncated log file."
+            required: true
+          - label: "I attached the backtrace in the case of a crash."
+            required: true
```

```suggestion
            required: true
            - label: "I've tested with the latest mpv version to validate if the issue is not already fixed."
            required: true
            - label: "I provided all required information including system and mpv version."
            required: true
```

test with latest version might be optional, because not all users will be able to do so it they use packaged version in Ubuntu LTS form 5 years ago. But I want to mention it so they at least try.